object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,22 @@
+/* exported omit */
+
+// We need a storage for the new object.
+// Store each property value pair of source.
+// Delete properties that are included in the keys array.
+// Return the storage.
+
+function omit(source, keys) {
+  var newObject = {};
+  for (var property in source) {
+    for (var i = 0; i < keys.length; i++) {
+      var value = source[property];
+      newObject[property] = value;
+      delete newObject[keys[i]];
+    }
```

Comment:
I believe you can do this loop to delete properties after the first loop to create the properties runs. As of right now, you are running this full loop for every property in the object as you create it, and that is a lot of extra operations for the computer.